bentoml/_internal/bento/build_config.py

```diff
@@ -58,13 +58,15 @@ def _convert_python_version(py_version: t.Optional[str]) -> t.Optional[str]:
 @attr.frozen
 class DockerOptions:
     # Options for choosing a BentoML built-in docker images
-    distro: t.Optional[str] = attr.ib(
-        validator=attr.validators.optional(
-            attr.validators.in_(DOCKER_SUPPORTED_DISTROS)
-        ),
-        default=None,
+    distro: "t.Literal['slim', 'amazonlinux2', 'alpine', 'centos7', 'centos8']" = (
```

Comment:
this should be `t.Optional[t.Literal[...]]` ?

Reply:
Yep.